common/versions.yml

```diff
@@ -0,0 +1,6 @@
+# This file is generated by https://github.com/gpii-ops/gpii-version-updater.
+---
+
+gpii-flowmanager: "gpii/universal@sha256:c8ac9f5b586e325e128790564840807f25089b03fcbea7b2ac1086a9dd70f75c"
```

Comment:
I would discard `gpii-` prefix from `gpii-dataloader` instead of adding it to `flowmanager` and `preferences`.